src/Schema/Events/index.ts

```diff
@@ -579,6 +581,10 @@ export enum ActionType {
    * Corresponds to {@link ClickedOnPriceDisplayDropdown}
    */
   clickedOnPriceDisplayDropdown = "clickedOnPriceDisplayDropdown",
+  /**
+   * Corresponds to {@link ClickedPublish}
+   */
+  clickedPublish = "clickedPublish",
```

Comment:
I'm actually not sure if this is correct. Looking at the [spreadsheet](https://docs.google.com/spreadsheets/d/124_JBqIvgUGB7t3XcJeEew2Bvd624Y6qyQgYxK0V-3E/edit#gid=454969903) referenced in the ticket, I think this may need to be a different value?